README.md

```diff
@@ -23,9 +23,11 @@ for a guide.
 
 
 You can install the core requirements with `pip` (make sure this is
-the `pip` in your virtual environment though)
+the `pip` in your virtual environment though). 
```

Comment:
don't much care for this sentence. more accurate would be to note that you should

1. have setup a virtualenv
2. have source .../bin/activated it

Reply:
Good suggestion, expanded out README with virtualenv directions